### PR TITLE
Mapping deleted record are exported

### LINKED ISSUE #24455: Camunda exporters support mapping deleted events

### PR DESCRIPTION
**Pull request by nicpuppa**
## Description

This PR adds the support to export mapping deleted records

## Related issues

closes #24455 

blocked by: https://github.com/camunda/camunda/pull/24626

### CODE REVIEW

**Review by remcowesterhoud — COMMENTED** [review]
Looks similar to user delete, so I guess it's correct 🤷 
I won't approve it, I'm lacking too much knowledge about this exporter atm so I'll leave that to someone else.

**Review by Ben-Sheppard — APPROVED** [review]
LGTM

**Review by koevskinikola — APPROVED** [review]
LGTM @nicpuppa 👍

**Pull request merged**
